### PR TITLE
Lance updates and raw_text replacement for text fields.

### PR DESCRIPTION
**Pull request by Karnor**
Changed text fields to raw_text fields where raw text is used to comply with the warnings in the error log.

Changed DefaultOnFaithClick to OpenGameViewData and DefaultOnCultureClick to OpenGameViewData as per Lance updates.

Added a error suppression effect/event combo to remove anal CK3 warning messages from the error log.

### CODE REVIEW

File: Advanced Character Search/common/scripted_effects/acs_error_suppression_effects.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿acs_suppress_error_effect = {
```
Comment (Vasolik) [inline Advanced Character Search/common/scripted_effects/acs_error_suppression_effects.txt]: can you just add prefixes to name of files and on script effects. I know that I was not consistent with it at beggining, but later I realized that I had a problem to find out what was an effect and what was a value for example.

So for script effects I think I used acs_se prefix.

I not sure if I have had an event but if not use acs_evt prefix.

**Comment by Vasolik** [comment]
Can you just do this renaming. I will try to publish this during weekend.

**Comment by Karnor** [comment]
Done.

File: Advanced Character Search/events/acs_evt_errors_suppression_event.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿namespace = acs_evt_errors_suppression_event
+
+acs_evt_errors_suppression_event.0001 = {
+	hidden = yes
+	orphan = yes
+
+	immediate = {
+		hidden_effect = {
+			acs_suppress_error_effect = { ERROR = acs_first_time_setup_v9 }
```
Comment (Vasolik) [inline Advanced Character Search/events/acs_evt_errors_suppression_event.txt]: I guess you need to update renamed scripted effect usage as well. :)

**Comment by Karnor** [comment]
Done.

**Pull request merged**